velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -166,7 +183,7 @@ class VectorFuzzer {
     opts_ = options;
   }
 
-  const VectorFuzzer::Options& getOptions() {
+  VectorFuzzer::Options& getOptions() {
```

Comment:
nit: Let's keep this returning `const VectorFuzzer::Options` and add another interface of `VectorFuzzer::Options& getMultableOptions()` to make the potential side-effect explicit.